ultradns-rest/src/main/java/denominator/ultradns/model/GeoInfo.java

```diff
@@ -0,0 +1,25 @@
+package denominator.ultradns.model;
+
+import java.util.TreeSet;
+
+public class GeoInfo {
+
+    private String name;
+    private TreeSet<String> codes = new TreeSet<String>();
```

Comment:
We got the ultra_rest_api-api-XXX.jar where all the DTO are declared. There it was mentioned as TreeSet, so used the same